setup_linux.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Linux Setup"
+author:
+  name: "Thomas Roppelt"
+  affiliation: "Friedrich-Alexander-Universität Erlangen-Nürnberg (FAU)"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+bibliography: library.bib
+---
+
+```{r setup, echo = FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+## Linux distributions (distros)
+Looking at Linux distributions and deciding for one can be a daunting task for newcomers.
+Here are some key differences between distributions:
+
+### Release Model
```

Comment:
I'm being a bit picky here but the newlines around the headings are a bit inconsistent.

I think the proper way is

```
blah blah


## heading

foo bar
```